pipeline/frontend/yaml/compiler/settings/params_test.go

```diff
@@ -59,6 +59,11 @@ func TestParamsToEnv(t *testing.T) {
 	got := map[string]string{}
 	assert.NoError(t, ParamsToEnv(from, got, secrets))
 	assert.EqualValues(t, want, got, "Problem converting plugin parameters to environment variables")
+
+	// handle edge cases (#1609)
+	got = map[string]string{}
+	assert.NoError(t, ParamsToEnv(map[string]interface{}{"a": []interface{}{"a", nil}}, got, nil))
```

Comment:
@6543 So the yaml for that would look like this 🤔 :

```yml
a:
 a:
```

Reply:
well yes ... it's obvious wrong ... but it should not be able to crash the server

Reply:
Yeah sure, just wanted to understand how it would look like. Could you maybe add the yaml as a comment as well.

Reply:
no I already reverenced the issue id ... and it's nothing to be encouraged to do

else we would also mention miss use examples elsewhere too, we just check and if there was an issue link to it